### DISCUSSION TITLE
Possibility of VLESS VPN profile support in Gluetun?

### DISCUSSION THREAD

**Discussion by woundwoort** [qdm12/gluetun#2397] [Ideas]
I hope this message finds you well. I have been using Gluetun for a while now and appreciate the robust support it offers for various VPN protocols. I am currently interested in using the VLESS protocol, which is known for its improved performance and security compared to other protocols.

Given the increasing popularity of VLESS, are there any plans to support VLESS VPN profiles in Gluetun in the near future? If not, could you please provide some insight into the challenges or limitations that might prevent its implementation?

Thank you for your time and the continuous development of this valuable project.

**Comment by zengateway** [comment]
+1

**Comment by LionMilk** [comment]
+1

**Comment by strunzoz** [comment]
like this?

https://github.com/f-normies/glueless/

**Comment by vrcher** [comment]
+1
